types/spinnies/index.d.ts

```diff
@@ -0,0 +1,150 @@
+// Type definitions for spinnies 0.5
+// Project: https://github.com/jcarpanelli/spinnies
+// Definitions by:  Adam Jarret <https://github.com/adamjarret>
```

Comment:
I understand this can be an attribution, but we strongly advice to get explicit ACK of
the person listed here, as kind of best-practise (the person will be target of notifications)

Reply:
@peterblazejewicz given that I don't have an explicit ACK from @adamjarret here, but he is (as best as I can tell) the original author of the types in question, what would you suggest would be appropriate attribution?

Perhaps I could just include an attribution comment below the header, assuming that style checks allow for that?

Reply:
Addressed as I proposed in my previous comment. Please resolve if you feel this is an appropriate resolution.

Reply:
Yes, thanks for taking this into consideration!